src/Encamina.Enmarcha.SemanticKernel.Plugins.Memory/Encamina.Enmarcha.SemanticKernel.Plugins.Memory.csproj

```diff
@@ -7,19 +7,19 @@
   </PropertyGroup>
 
   <PropertyGroup Condition="'$(Configuration)|$(Platform)'=='Debug|AnyCPU'">
-    <NoWarn>1701;1702;SKEXP0003</NoWarn>
+    <NoWarn>1701;1702;SKEXP0001;SKEXP0003</NoWarn>
```

Comment:
Ídem.

Check this exceptions, I don't know if all of them are still valid or required since the SK release notes indicates the following → «_Reduce number of experimental flags_»

This means that perhaps some exceptions we had are not required anymore. 

Please verify this in every project that has been affected.

Reply:
Remove `SKEXP0003` exception.